apps/_sandbox/.env.sample

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_MAINNET_URI=
+NEXT_PUBLIC_SERVER_MAINNET_URI=
```

Comment:
Don't need a server mainnet uri, we're not going to be publishing this to vercel anytime soon